cores/esp32/Arduino.h

```diff
@@ -127,7 +127,8 @@ void loop(void);
 
 long random(long, long);
 void randomSeed(unsigned long);
-long map(long, long, long, long, long);
+long Arduino_map(long, long, long, long, long);
+#define map Arduino_map
 
```

Comment:
I think it's better to define it as ```map(a,b,c,d,e)``` so you can per say have class members named ```map```